shared-data/default-theme/html/jsapi/templates/modal-display-keybindings.html

```diff
@@ -0,0 +1,25 @@
+<div class="modal-dialog">
+  <div class="modal-content">
+    <div class="modal-header">
+      <button type="button" class="close button-info" data-dismiss="modal" aria-hidden="true">&times;</button>
+      <h4 class="modal-title">Keybindings</h4>
+    </div>
+    <div class="modal-body modal-body-light-gray">
+      <table class="default-table">
+        <thead>
+          <th>Title</th>
+          <th>Binding</th>
+        </thead>
+        <tbody>
+          <% _.each(keybindings, function(binding) { %>
+            <tr>
+              <td><%= binding.title %></td>
+              <td><%= binding.keys %></td>
+            </tr>
+          <% }); %>
+        </tbody>
+      </table>
+      <p>Keybindings are based on <a href="https://support.google.com/mail/answer/6594?hl=en" target="_blank">Gmails keybindings</a>.</p>
```

Comment:
There is a typo here - It should say Gmail's (with the apostrophe).

Also, the message itself, as well as the Keybindings, Title and Binding headings need to be translatable.

Accommodating i18n may mean rephrasing things slightly, because the sentence should be translated as a single unit (translating fragments leads to poor translations due to grammatical differences between languages), while the link markup should not be included in the translated data (so translators don't introduce typoes that break the layout/url). This actually gets awkward enough that it may be best to just skip the attribution entirely.